src/ontology/imports/merged_import.owl

```diff
@@ -224634,7 +224634,6 @@ EquivalentClasses(<http://purl.obolibrary.org/obo/UBERON_8450002> ObjectIntersec
 # Class: <http://purl.obolibrary.org/obo/UBERON_8470000> (placental blood)
 
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://orcid.org/0000-0001-6595-0902") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.ncbi.nlm.nih.gov/books/NBK53254/") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/UBERON_8470000> "Blood located in the placenta.")
-AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/UBERON_8470000> <https://orcid.org/0000-0002-1773-2692>)
```

Comment:
Shouldn't this be deleted in Uberon first, and it will be updated once we update the imports?